server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGateConditionsUpdater.java

```diff
@@ -186,10 +186,14 @@ private static void checkConditionDoesNotAlreadyExistOnSameMetricAndPeriod(Colle
     }
   }
 
-  private static void checkRatingMetric(MetricDto metric, String operator, @Nullable String warningThreshold, @Nullable String errorThreshold, Errors errors) {
+  private static void checkRatingMetric(MetricDto metric, String operator, @Nullable String warningThreshold, @Nullable String errorThreshold, @Nullable Integer period,
+    Errors errors) {
     if (!metric.getValueType().equals(RATING.name())) {
       return;
     }
+    if (period != null && !metric.getKey().startsWith("new_")) {
```

Comment:
I'm not sure I understand. Shouldn't the `new_*` metrics be always on the leak period?